src/components/Camera/style.css

```diff
@@ -103,6 +102,21 @@
   /*Mirroring transform: scale(-1, 1); */
 }
 
+@supports (object-fit: cover){
+  .video{height: 100%; width:100%}
+}
+/* this will only apply styles to Edge */
+@supports (object-fit: cover) and (-ms-ime-align:auto) {
```

Comment:
hm, I see, so is the object-fit really necessary? it seems we are just trying to detect if it's IE by checking the `-ms-ime-align` property, right?

Reply:
does IE 11 have the same problem?

Reply:
We could remove `object-fit: cover` because IE11 doesn't not support `getUserMedia` so we wouldn't see the issue there, but this feature query is specific to target Edge and not IE11.

Reply:
got it